docs/README.rst

```diff
@@ -341,7 +341,7 @@ If you don't define ``bindAddr`` in the Frontend configuration, you must include
 The Controller watches for the annotation key ``virtual-server.f5.com/ip``.
 This annotation must contain the IP address you want to assign to the virtual server.
 
-- You can `configure an IPAM system </containers/latest/kubernetes/ktclr-manage-bigip-objects.html#use-ipam-to-assign-ip-addresses-to-big-ip-LTM-virtual-servers>`_ to write out an annotation containing the selected IP address.
+- You can `configure an IPAM system <http://clouddocs.f5.com/containers/v1/kubernetes/kctlr-manage-bigip-objects.html#use-ipam-to-assign-ip-addresses-to-big-ip-ltm-virtual-servers>`_ to write out an annotation containing the selected IP address.
```

Comment:
I believe if you changed 'LTM' to 'ltm', that should be sufficient to fix this link.We should avoid adding the full path to the docs and not hardcode and version dependencies (latest should work just fine).